lib/matplotlib/tests/test_path.py

```diff
@@ -102,6 +101,50 @@ def test_exact_extents(path, extents):
     assert np.all(path.get_extents().extents == extents)
 
 
+@image_comparison(['stroked_bbox'], remove_text=True,
+                  extensions=['pdf', 'svg', 'png'])
```

Comment:
These are the default extensions?
```suggestion
@image_comparison(['stroked_bbox'], remove_text=True)
```